nfv-example-cnf-catalog.Dockerfile

```diff
@@ -0,0 +1,11 @@
+# The base image is expected to contain
+# /bin/opm (with a serve subcommand) and /bin/grpc_health_probe
+FROM registry.redhat.io/openshift4/ose-operator-registry:v4.9
```

Comment:
This registry requires creds, do you know if this is required or if there's any other option? otherwise we will need some creds in here to build

Reply:
Good point @tonyskapunk. I looked a bit and found that the upstream-opm-builder image has the required binaries and is 4x times smaller. 

Confirmed that the index is support queries using the grpc API.